showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITUnaryCallable.java

```diff
@@ -100,11 +100,30 @@ public void testGrpc_shutdown() {
   }
 
   @Test
-  public void testHttpJson() {
+  public void testHttpJson_receiveContent() {
     assertThat(echoHttpJson("http-echo?")).isEqualTo("http-echo?");
     assertThat(echoHttpJson("http-echo!")).isEqualTo("http-echo!");
   }
 
+  @Test
+  public void testHttpJson_serverResponseError_throwsException() {
+    StatusCode.Code cancelledStatusCode = StatusCode.Code.CANCELLED;
+    EchoRequest requestWithServerError =
+        EchoRequest.newBuilder()
+            .setError(Status.newBuilder().setCode(cancelledStatusCode.ordinal()).build())
+            .build();
+    CancelledException exception =
+        assertThrows(CancelledException.class, () -> httpJsonClient.echo(requestWithServerError));
+    assertThat(exception.getStatusCode().getCode()).isEqualTo(cancelledStatusCode);
+  }
```

Comment:
nit: How about we use `HttpJsonStatusCode.Code.CANCELLED` here?

Reply:
Sounds good. Updating.